examples/examples3d.jl

```diff
@@ -625,8 +625,61 @@
         streamplot(f, -1.5..1.5, -1.5..1.5, -1.5..1.5, colormap = :magma, gridsize = (10, 10), arrow_size = 0.06)
     end
 
+    @cell "Fractional Brownian surface" ["3d"] begin
+        # This example was provided by Moritz Schauer (@mschauer).
+        using SparseArrays, LinearAlgebra
+
+        # Here, we define some form of noise function:
+        function gridlaplacian(m, n)
+            S = sparse(0.0I, n*m, n*m)
+            linear = LinearIndices((1:m, 1:n))
+            for i in 1:m
+                for j in 1:n
+                    for (i2, j2) in ((i + 1, j), (i, j + 1))
+                        if i2 <= m && j2 <= n
+                            S[linear[i, j], linear[i2, j2]] -= 1
+                            S[linear[i2, j2], linear[i, j]] -= 1
+                            S[linear[i, j], linear[i, j]] += 1
+                            S[linear[i2, j2], linear[i2, j2]] += 1
+                        end
+                    end
+                end
+            end
+            return S
+        end
+
+        # d is used to denote the size of the data
+        d = 150
+
+        data = 0.1randn(d,d) + reshape(
+                cholesky(gridlaplacian(d,d) + 0.003I) \ randn(d*d),
+                d, d
+        )
+
+        surface(data; shading=false, show_axis=false, colormap = :deep)
+    end
+
+    @cell "Coloured fractional Brownian noise field" ["3d"] begin
+        # This example was contributed by Harmen Stoppels (@haampie)
+        
+        using FFTW
+
+        function cloud(n = 256, p = 0.75f0)
```

Comment:
# Obtain approximately fractal Brownian appropriately damping the high frequencies of Fourier transformed 
        # spatial white noise and (inverse) Fourier transform the result back into the spatial domain